VIDEO_1_SCRIPT.md

```diff
@@ -0,0 +1,97 @@
+# Video 1: What are APIs
+
+- What are APIs
+- Brief history of APIs (when and how APIs were born)
+- Types of APIs
+- Benefits of APIs
+
+Hello! My name is … and today I'm going to talk about APIs. Maybe you've heard of APIs before because they're in almost all the technology we use in our everyday lives.
+
+We're going to dive a little bit into their history and origins, explaining what they are, what types of APIs exist, and what are their benefits. Finally, we will show you some API examples to make the concept and its use clearer.
+
+Let’s go!
+
+## ***What are APIs***
+
+***API is the acronym for Application Programming Interface. APIs are a tool that facilitates the communication and relationship between several applications. It’s the way apps are able to exchange messages or data and understand each other.***
+
+APIs allow your products and services to communicate with others, without needing to know how they are implemented. An API is a set of definitions and protocols used to develop and integrate application software.
+
+For communicating we need to be on the same page; it's the same for machines. You see, API is an user interface that enables interaction and communication between software and a person.
```

Comment:
Won't it sound better if we rephrase :

*You see, an API is a user interface that enables interaction and communication between software and a person*

Reply:
Yes, it sounds better this way.

As this PR is closed I made the change in the new script [file](https://github.com/Barbanio/training/blob/master/VIDEO_1_SCRIPT:_WHAT_ARE_APIS.md). 

Thanks for your feedback @thulieblack 🙂